src/main/java/net/minecraftforge/event/entity/player/PlayerInteractEvent.java

```diff
@@ -45,11 +46,12 @@
     public final World world;
     public final BlockPos pos;
     public final EnumFacing face; // Can be null if unknown
+    public final Vec3 localPos; // Can be null if unknown
 
     public Result useBlock = DEFAULT;
     public Result useItem = DEFAULT;
 
-    public PlayerInteractEvent(EntityPlayer player, Action action, BlockPos pos, EnumFacing face, World world)
+    public PlayerInteractEvent(EntityPlayer player, Action action, BlockPos pos, EnumFacing face, World world, Vec3 localPos)
     {
```

Comment:
Oh, sorry.. Was having a mental blank and forgot Java supported constructor overloading.